packages/react/test/specs/lib/getKeyDownHandlers-test.ts

```diff
@@ -142,17 +185,35 @@ describe('getKeyDownHandlers', () => {
       const keyHandlers = getKeyDownHandlers(actions, actionsDefinition)
       expect(keyHandlers.hasOwnProperty(partElementName)).toBeFalsy()
     })
-    test("when acessibility's actionsDefinition is null", () => {
+
+    test("when accessibility's actionsDefinition is null", () => {
       const actions = { otherAction: () => {} }
       const keyHandlers = getKeyDownHandlers(actions, null)
 
       expect(keyHandlers.hasOwnProperty(partElementName)).toBeFalsy()
     })
+
     test('there are not common actions and actions definition', () => {
       const actions = { otherAction: () => {} }
       const keyHandlers = getKeyDownHandlers(actions, actionsDefinition)
 
       expect(keyHandlers.hasOwnProperty(partElementName)).toBeFalsy()
     })
+
+    test('when action definition has no keyCombinations', () => {
+      const actions = {
+        testAction: () => {},
+        otherAction: () => {},
+      }
+
+      actionsDefinition.anotherPart = {
+        otherAction: {
+          keyCombinations: false,
```

Comment:
Can we add tests for other falsy values as well, like undefined, null, [] etc.

Reply:
added test, fixed code for []. could you please recheck?

Reply:
Looks good!